code/game/objects/structures/transit_tubes/station.dm

```diff
@@ -147,7 +147,7 @@
 		sleep(OPEN_DURATION + 2)
 		pod_moving = 0
 		if(!QDELETED(pod))
-			pod.air_contents.share(loc.return_air()) //mix the pod's gas mixture with the tile it's on
+			pod.air_contents.share(loc.return_air(), 1) //mix the pod's gas mixture with the tile it's on
```

Comment:
This argument indicates the number of adjacent turfs i.e. the number of turfs that will be share()ed with. Since pods only share with the turf directly beneath them, I set it to 1.